psq/queue.py

```diff
@@ -91,7 +95,12 @@ def enqueue_task(self, task):
         Note that this does not store the task.
         """
         data = dumps(task)
-        self.topic.publish(data)
+
+        if self.async:
+            self.topic.publish(data)
+        else:
+            task.execute(queue=self)
```

Comment:
Oops, yes. I've added measure_time() as well and made the logging distinctive depending on async.